README.md

```diff
@@ -103,6 +103,7 @@ https://spdx.org/licenses
 | [**mdbook-linkcheck**](https://github.com/Michael-F-Bryan/mdbook-linkcheck) | `$CARGO_HOME/bin` | [GitHub Releases](https://github.com/Michael-F-Bryan/mdbook-linkcheck/releases) | Linux, macOS, Windows | [MIT](https://github.com/Michael-F-Bryan/mdbook-linkcheck/blob/HEAD/LICENSE) |
 | [**mdbook**](https://github.com/rust-lang/mdBook) | `$CARGO_HOME/bin` | [GitHub Releases](https://github.com/rust-lang/mdBook/releases) | Linux, macOS, Windows | [MPL-2.0](https://github.com/rust-lang/mdBook/blob/HEAD/LICENSE) |
 | [**nextest**](https://github.com/nextest-rs/nextest) (alias: `cargo-nextest`) | `$CARGO_HOME/bin` | cargo-binstall | Linux, macOS, Windows | [Apache-2.0](https://github.com/nextest-rs/nextest/blob/HEAD/LICENSE-APACHE) OR [MIT](https://github.com/nextest-rs/nextest/blob/HEAD/LICENSE-MIT) |
+| [**osv-scanner**]( https://github.com/google/osv-scanner) | `$HOME/.install-action/bin` | [GitHub Releases]( https://github.com/google/osv-scanner/releases) | Linux, macOS, Windows | [Apache-2](https://github.com/google/osv-scanner/blob/main/LICENSE) |
```

Comment:
I copied `$HOME/.install-action/bin` from the `protoc` entry .. but I see other non-Rust entries (shellcheck,shfmt,syft) in README.md put `/usr/local/bin` in that column.

I see in `main.sh` that it says
> Copying files to /usr/local/include requires sudo, so do not use it.

Reply:
`protoc` is special. It needs to copy more files other than `protoc` binary itself.
In the case of no such special requirements, I would prefer to use `/usr/local/bin` for non-Rust tools.

Reply:
Is this ok @taiki-e , or should I use `/usr/local/bin` ?